### DISCUSSION TITLE
Q: How to change resolution/qualitiy

### DISCUSSION THREAD

**Discussion by PulsarFX** [regseb/castkodi#55] [General]
How can I change the quality of youtube videos that are played to kodi? Currently it plays 640x360 on a 1080p screen (HDTV) no matter how I start it from firefox.

**Comment by PulsarFX** [comment]
Today it works. :) 
I don't know if it is because I changed some settings in the youtube addon in Kodi or if it is video related (both videos I tested support 1080p in YT)

**Comment by regseb** [comment]
@maulwuff _Cast Kodi_ only sends the video ID to the [YouTube add-on in Kodi](https://github.com/jdf76/plugin.video.youtube). _Video quality_ can be configured in the _Settings_ of the YouTube add-on in Kodi.
